app/models/preserved_copy.rb

```diff
@@ -8,23 +8,30 @@ class PreservedCopy < ApplicationRecord
   UNEXPECTED_VERSION_ON_STORAGE_STATUS = 'unexpected_version_on_storage'.freeze
   VALIDITY_UNKNOWN_STATUS = 'validity_unknown'.freeze
   UNREPLICATED_STATUS = 'unreplicated'.freeze
+  FILE_NOT_FOUND = 'file_not_found'.freeze
+  CHECKSUM_MISMATCH = 'checksum_mismatch'.freeze
```

Comment:
maybe rename this? db_s3_checksum_mismatch? i dunno..

Reply:
This is one of those things that will only apply to archive PCs when that model is split out.  In that context, I think the meaning of the status is a bit clearer, but it will still be sorta wrong to think about it at this level.  The zip **part** is what will be not found or mismatched.  The APC might have parts not found and mismatched simultaneously.

Not a blocker, but worth other people on the team weighing in on, maybe not worth adding now if we know we have to nuke it soon.

Reply:
Yeah, i have note in this PR, that when the model is split out to fix the statuses. But yeah any thoughts on renaming?

Reply:
The names themselves are OK.